index.html

```diff
@@ -38,5 +38,123 @@ <h4 class="connect">LET'S CONNECT</h4>
+        <div class="card">
+            <img class="cardsection" src="images/forthwork.png" alt="SnapshootPortfolio">
+            <div class="cardsection">
+                <h4>Multi-Post Stories</h4> 
+                <ul class="client">
+                    <li class="worktime"><b>CANOPY</b></li>
+                    <li class="worktime"><img src="images/Counter.png"></li>
+                    <li class="worktime worktitle">Back End Dev</li>
+                    <li class="worktime"><img src="images/Counter.png"></li>
+                    <li class="worktitle">2015</li>
+                </ul>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="client">
+                    <li class="worktime tag"><b>html</b></li>
+                    <li class="worktime tag"><b>css</b></li>
+                    <li class="worktime tag"><b>javaScript</b></li>
+                </ul>
+                <button class="cardButton" type="button">See Project</button>
+            </div>
+        </div>
+    </section>
+    <section class="aboutMe">
+        <h2>About <br>Myself</h2>
+        <p class="resume">Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+        <h4 class="connect">LET'S CONNECT</h4>
+        <div class="social">
+            <ul class="personalMedia">
+                <li><a class="media" href="https://twitter.com/Santiag24209785"><img src="images/twitter.png"></a></li>
+                <li><a class="media" href="https://www.linkedin.com/in/santiago-munoz-0b2b1a260/"><img src="images/linkedin.png"></a></li>
+                <li><a class="media" href="https://github.com/smunoz1988"><img src="images/github.png"></a></li>
+            </ul>
+        </div>
+        <button class="resumeButton" type="button">Get my resume</button>
+        <div>
+            <div class="categories">
+                <div class="container">
+                    <p class="atribute">Languages</p>
+                    <img class="arrow" src="images/arrowD.png">
+                </div>
+                <ul class="languaje">
+                    <li class="js"><img class="lang" src="images/js.png"><p>JavaScript</p></li>
+                    <li class="html"><img class="lang" src="images/html.png"><p>HTML</p></li>
+                    <li class="css"><img class="lang" src="images/css.png"><p>CSS</p></li>
+                </ul>
+                <div class="container">
+                    <p class="atribute">Frameworks</p>
+                    <img class="arrow" src="images/arrowR.png">
+                </div>
+                <hr>
+                <div class="container">
+                    <p class="atribute">Skills</p>
+                    <img class="arrow" src="images/arrowr.png">
+                </div>
```

Comment:
Well done, only one small thing fix the typo in the link :

![arrow typo](https://user-images.githubusercontent.com/59707859/219804755-4e6bbfd6-db0c-4178-bc30-efbef50d7954.PNG)